movie.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+require_relative 'source'
+
+class Movie < Item
+  attr_accessor :silent
+
+  def initialize(publish_date, silent)
+    super(publish_date)
+    @silent = silent
+  end
```

Comment:
Remember that the `Item` class takes more than just the `publish_date` variable as its parameters. I suggest you to leave it like this:
```suggestion
  def initialize(silent, publish_date, archived, id = Time.now.to_f.to_s)
    super(publish_date, archived, id)
    @silent = silent
  end
```

Reply:
Done!